lib/paths.js

```diff
@@ -14,17 +14,17 @@ const {
 
 export const getPaths = ({ cacheRoot, stateRoot }) => ({
   secrets: join(stateRoot, 'secrets'),
-  moduleCache: join(cacheRoot, 'modules'),
-  moduleState: join(stateRoot, 'modules'),
-  moduleSourcesDir: join(cacheRoot, 'sources'),
-  moduleVersionsDir: join(stateRoot, 'modules', 'latest'),
+  runtimeCache: join(cacheRoot, 'runtimes'),
+  runtimeState: join(stateRoot, 'runtimes'),
+  subnetSourcesDir: join(cacheRoot, 'subnets', 'sources'),
+  subnetVersionsDir: join(stateRoot, 'subnets', 'latest'),
   lockFile: join(stateRoot, '.lock')
 })
 
 export const getDefaultRootDirs = () => {
   switch (platform()) {
     case 'darwin': {
-      const appId = 'app.filstation.core'
+      const appId = 'network.checker.node'
```

Comment:
I believe this is a breaking change for existing users as it will change the location where Checker Node keeps the state, e.g. the number of jobs completed.

In other words, people running Checker Node via `station` CLI will see their job count drop to zero after the upgrade.

I think people running the node via Docker will be affected too, because the instructions in README mount the entire `/home/node/.local/state/` directory.

Are we fine with that?

Reply:
Indeed this is a breaking change, I pointed part of that out in https://github.com/CheckerNetwork/core/pull/677/files/57a25a2730b1658784b12261ae9b4606e34e780c#diff-5af41a7c365c40a642be45fcfff655a897ea5a92e384196ec65d4110f80f2714.

I hadn't realized it would also restart the local job count. Thank you for pointing that out!

Are we still concerned about per-instance job count?

If we were, I guess we would need to add migration logic, or documentation at least.

Reply:
Done :) It was a tiny bit more complex, since more paths changed, but very doable after all